http/servlet-undertow/src/test/java/io/quarkus/ts/http/undertow/HttpServletWithSessionListenerIT.java

```diff
@@ -51,6 +57,30 @@ public void sessionSecured() {
         thenCheckActiveSessionsEqualTo(1);
     }
 
+    @Test
+    @DisplayName("undertow servlet test http2")
+    public void underTowHttp2Servert(VertxTestContext vertxTestContext) {
```

Comment:
- Undertow shouldn't be written as `underTow` as it suggests it is compound of 2 words
- `Servert` is probably a typo